build/Rulesets/Roslyn.ruleset

```diff
@@ -144,4 +144,9 @@
     <Rule Id="IDE1006" Action="Error" />                      <!-- Naming styles                                Task Open()                                         Task OpenAsync()                      -->
     <Rule Id="IDE1006WithoutSuggestion" Action="Info" />
   </Rules>
+  <Rules AnalyzerId="Microsoft.VisualStudio.Threading.Analyzers" RuleNamespace="Microsoft.VisualStudio.Threading.Analyzers">
+    <Rule Id="VSTHRD200" Action="None" />                       <!-- Naming styles                              Task Open()                                         Task OpenAsync()                      -->
```

Comment:
FYI this isn't just about style (although that's worth something). This also enables other analyzers to do their job. For instance, if you're in an async method and you call Open when an OpenAsync method is available, you're probably doing the wrong thing and will end up blocking a thread needlessly. But the analyzer that catches that (VSTHRD103) only works if you follow the Async suffix naming convention.